test/test_amity.py

```diff
@@ -0,0 +1,415 @@
+import unittest
+from office_space_allocation import amity, person, fellow, staff, room, office, livingroom
+from office_space_allocation.utilities import InvalidRoomOccupantError, RoomFullError
+from office_space_allocation import db
+
+class TestAmityClassStructure(unittest.TestCase):
+    """
+    Tests the class structure of the project
+    """
+
+    def test_can_create_amity_instance(self):
+        """
+        Should be able to create an instance of the Amity class
+        """
+        self.amity = amity.Amity()
+        self.assertIsInstance(self.amity, amity.Amity)
```

Comment:
Too obvious tests. You just declared an instance of amity, there is no need to test if the instance you just created is an instance of what you created